news/532.feature

```diff
@@ -0,0 +1,3 @@
+Added one new content API helper function:
+- `api.content.get_path`: Get path of an object. Optionally receive absolute path from Zope root.
```

Comment:
This can all be merged onto one line. Also the change log file is in reStructuredText, not Markdown, so inline literals need double backticks. Finally, I'm not certain of how these display in the change log or if there is any processing of list items, and it looks kind of bad. See example under the Documentation section.

https://github.com/plone/plone.api/blob/main/CHANGES.rst#225-2025-01-24

```suggestion
```